source/administration/upgrade.rst

```diff
@@ -84,6 +84,8 @@ Location of your local storage directory
    The special directories within mattermost are ``config``, ``logs``, ``plugins``, ``client/plugins``, and ``data`` (unless you have a different value configured for local storage, as per *Before you begin*). The following command clears the contents of mattermost, preserving only those directories and their contents.
    You should first modify the last part to ``xargs echo rm -r`` to verify what will be executed.
 
+   If you store TLSCert/TLSKey files or other information within your `/opt/mattermost` folder you will want to add `-o -path mattermost/yourFolderHere` to the below command or you will have to manually copy the from the backup into the new install.
```

Comment:
```suggestion
   If you store TLSCert/TLSKey files or other information within your ``/opt/mattermost`` folder, you should add ``-o -path mattermost/yourFolderHere`` to the below command or you will have to manually copy the from the backup into the new install.
```

Also, it sounds like there is a missing word in "you will have to manually copy the from the backup into the new install".